### PR TITLE
Tutorial update.

### PR DESCRIPTION
**Pull request by illia-stv**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Basic and advanced tutorial update. Closes #000.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-1.md [hunk 1]
```diff
@@ -18,29 +18,34 @@ If you want to see the final product of this tutorial before you plunge in, chec
...
 
-The editor has already been created in the `app.js` file with some basic plugins. All you need to do is clone the repository, navigate to the [starter-files directory](https://github.com/ckeditor/ckeditor5-tutorials-examples/tree/main/abbreviation-plugin/starter-files), run the `npm install` command, and you can start coding right away.
+The editor has already been created in the `main.js` file with some basic plugins. All you need to do is clone the repository, navigate to the [starter-files directory](https://github.com/ckeditor/ckeditor5-tutorials-examples/tree/main/abbreviation-plugin/starter-files), run the `npm install` command, and you can start coding right away.
 
-The webpack is also already configured, so you can just use the `npm run build` command to build your application. Whenever you want to check anything in the browser, save the changes and run build again. Then, refresh the page in your browser (remember to turn off caching, so that new changes are displayed instantly).
+You can just use the `npm run dev` command to run your application. Whenever you want to check anything in the browser, save the changes and run application again.
```
Comment (Witoso) [inline docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-1.md:25]: Code block is worth thousands of words IMHO

```shell
git clone https://github.com/ckeditor/ckeditor5-tutorials-examples;
cd ckeditor5-tutorials-examples/abbreviation-plugin/starter-files;

npm install;
npm run dev;
```

Code block is noticeable, and copyable. And we need to clearly promote it.

**Review by Witoso — CHANGES_REQUESTED** [review]

**Review by godai78 — APPROVED** [review]

File: docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ If you want to see the final product of this tutorial before you plunge in, chec
 
 ## Improving accessibility
 
-First, we make our plugin accessible for users who rely on keyboards for navigation. We want to ensure that pressing <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd> will move focus around in the form view, and pressing `esc` will close it.
+First, we make our plugin accessible for users who rely on keyboards for navigation. We want to ensure that pressing <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd> will move focus around in the form view, and pressing <kbd>Esc</kbd> will close it.
```
Comment (godai78) [inline docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md:19]: Are these keys behaving like this after recent changes?

Comment (gorzelinski) [reply docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md:19]: From my testing - everything works except the escape key. But I fixed it in the tutorial files.

**Review by Witoso — APPROVED** [review]
LGTM, I like extra touches, like `kbd` , `git clone`'s look good 👍

**Pull request merged**